.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### General information
+
+  * Minishift version: 
+  * Download from: github releases / brew / chocolatey
+  * OS: Linux / macOS / Windows
+  * OS version/distro: 
```

Comment:
The lines 6 to 7 are unnecessary IMO. Agree with @LalatenduMohanty  that `--show-libmachine-logs -v 5` will display which hypervisor user is using.

Reply:
@budhrg I guess there will be some issues where user do not need to run `--show-libmachine-logs -v5 `. So if user is giving out put of `--show-libmachine-logs -v5 ` then, user does not need to give these details.